src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -728,7 +728,6 @@ public static void GetFullPath_Windows_UNC_Invalid(string invalidPath)
         }
 
         [PlatformSpecific(TestPlatforms.Windows)]  // Uses P/Invokes to get short path name
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
Does this pass in netfx? As since ConditionalFact attribute is broken in desktop this test was not running, so we don't know if it actually passes. Would you mind trying that?